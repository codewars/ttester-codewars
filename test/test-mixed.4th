\ Copyright 2023 nomennescio

\ pass
<{ 1 2e -> 1 2e }>
<{ 1 1e 2 1e f+ -> 1 2 2e }>
<{ 1 2e 2 3e f* -> 1 2 6e }>
<{ 1 0e 2 1e 3 2e -> 1 2 3 0e 1e 2e }>
<{ 1 1e 1e f- 2 0e fcos 3 4e 2e f/ -> 1 2 3 0e 1e 2e }>

\ fail on number of results
<{ 1 2e dup -> 1 2e }>
<{ 1 2e fdup -> 1 2e }>
<{ 1 1e 2 1e f+ dup -> 1 2 2e }>
<{ 1 1e 2 1e f+ fdup -> 1 2 2e }>
<{ 1 2e 2 3e f* dup -> 1 2 6e }>
<{ 1 2e 2 3e f* fdup -> 1 2 6e }>
<{ 1 0e 2 1e 3 2e dup -> 1 2 3 0e 1e 2e }>
<{ 1 0e 2 1e 3 2e fdup -> 1 2 3 0e 1e 2e }>
<{ 1 1e 1e f- 2 0e fcos 3 4e 2e f/ dup -> 1 2 3 0e 1e 2e }>
<{ 1 1e 1e f- 2 0e fcos 3 4e 2e f/ fdup -> 1 2 3 0e 1e 2e }>

<{ 1 2e dup fdup -> 1 2e }>
<{ 1 2e fdup dup -> 1 2e }>
<{ 1 dup 2e fdup -> 1 2e }>

\ fail on differences in outcomes
<{ 1 2e -> 2 2e }>
<{ 1 2e -> 1 1e }>
<{ 1 2e -> 2 1e }>

<{ 1 2 3 4e 5e 6e -> 4 5 6 4e 5e 6e }>
<{ 1 2 3 4e 5e 6e -> 1 2 3 1e 2e 3e }>
<{ 1 2 3 4e 5e 6e -> 4 5 6 1e 2e 3e }>