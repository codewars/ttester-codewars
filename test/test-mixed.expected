
<PASSED::>Test Passed

<PASSED::>Test Passed

<PASSED::>Test Passed

<PASSED::>Test Passed

<PASSED::>Test Passed

<FAILED::>Wrong number of cell results, expected 1 , got 2 

<FAILED::>Wrong number of float results, expected 1 , got 2 

<FAILED::>Wrong number of cell results, expected 2 , got 3 

<FAILED::>Wrong number of float results, expected 1 , got 2 

<FAILED::>Wrong number of cell results, expected 2 , got 3 

<FAILED::>Wrong number of float results, expected 1 , got 2 

<FAILED::>Wrong number of cell results, expected 3 , got 4 

<FAILED::>Wrong number of float results, expected 3 , got 4 

<FAILED::>Wrong number of cell results, expected 3 , got 4 

<FAILED::>Wrong number of float results, expected 3 , got 4 

<FAILED::>Wrong number of cell results, expected 1 , got 2 
<:LF:>Wrong number of float results, expected 1 , got 2 

<FAILED::>Wrong number of cell results, expected 1 , got 2 
<:LF:>Wrong number of float results, expected 1 , got 2 

<FAILED::>Wrong number of cell results, expected 1 , got 2 
<:LF:>Wrong number of float results, expected 1 , got 2 

<FAILED::>Expected 2 , got 1 

<FAILED::>Expected 1. , got 2. 

<FAILED::>Expected 2 , got 1 
<:LF:>Expected 1. , got 2. 

<FAILED::>Expected 4 5 6 , got 1 2 3 

<FAILED::>Expected 1. 2. 3. , got 4. 5. 6. 

<FAILED::>Expected 4 5 6 , got 1 2 3 
<:LF:>Expected 1. 2. 3. , got 4. 5. 6. 
